### PR TITLE
Add community post (Observable Infrastructure as Code)

### PR DESCRIPTION
**Pull request by mattstratton**
NOTE: as this is a syndicated community post, edits should be minor.

Signed-off-by: Matt Stratton <matt.stratton@gmail.com>

### CODE REVIEW

File: themes/default/content/blog/observable-infrastructure-as-code/index.md [hunk 1]
```diff
@@ -0,0 +1,416 @@
...
+Honeycomb uses events to figure out what's happening in your system. Suppose the events have special attributes, like `trace.trace_id` and `trace.span_id`, Honeycomb can correlate them. This way, it can tell you things like, "The trace for your request took 300ms and consisted of 100ms authentication and 200ms database access."
+
+So you can pump all your information to Honeycomb via events. Sometimes it doesn't need to be correlated; you only want to tell Honeycomb that your server uses 20% of memory. But in our case, we want to tie up all events related to one deployment into a trace, so we can figure out what we deploy, how long it takes, and if something goes wrong, what resource was responsible.
+
+[](#linking-up-pulumi-amp-honeycomb)Linking up Pulumi & Honeycomb
+------------------------------------------------------------------------------------------------------------------------------
+
+We need to initialize Libhoney and send the correct "Honeycomb event" for every "Pulumi event."
+
+But let's start with getting one event into Honeycomb. The pulumi up event.
```
Comment (nimbinatus) [inline themes/default/content/blog/observable-infrastructure-as-code/index.md:149]: Non-blocking
```suggestion
But let's start with getting one event into Honeycomb: the `pulumi up` event.
```

File: themes/default/content/blog/observable-infrastructure-as-code/index.md [hunk 2]
```diff
@@ -0,0 +1,416 @@
...
+This is what a trace could look like in Honeycomb:
+
+[![Example Trace](./example-trace.png)](./example-trace.png)
+
+Why not setting things up before I move deployment from my development machine to the cloud?
+
+This article will show you how to make your Pulumi deployment observable with [@honeycomb_io](https://dev.to/honeycomb_io)
+
+[](#prerequisites)Pre-Requisites
+---------------------------------------------------------------------------------------------
```
Comment (nimbinatus) [inline themes/default/content/blog/observable-infrastructure-as-code/index.md:35]: Our Hugo system adds the anchor links already, so this actually breaks the links (e.g., this one shows up as http://pulumi-hugo-origin-pr-987-40af68f2.s3-website.us-west-2.amazonaws.com/blog/observable-infrastructure-as-code/#prerequisitespre-requisites).
```suggestion
## Prerequisites
```

Do you want me to update all of them?

**Review by nimbinatus — CHANGES_REQUESTED** [review]
A non-blocking note, non-blocking grammar note or two, and a couple fixes for headers needed

File: themes/default/content/blog/observable-infrastructure-as-code/index.md (hunk 2 above)
Comment (mattstratton) [reply themes/default/content/blog/observable-infrastructure-as-code/index.md:35]: I'll fix the others after committing this one!

**Review by nimbinatus — APPROVED** [review]
+1. Only question is whether you've differentiated your paragraph at the beginning on any of the others. If you haven't, I'd be inclined to do a follow-up PR to either add a line underneath the paragraph or otherwise set it off so it's clearer when the community member's post begins. As is, though, I think this matches the others and is ready to go.

**Review by katcosgrove — APPROVED** [review]

**Pull request merged**